lib/views/pages/home/top_bar/app_top_bar.dart

```diff
@@ -21,7 +19,7 @@ class AppTopBar extends HookConsumerWidget {
       style: Theme.of(context).textTheme.headlineMedium,
     );
 
-    Widget userAvatar = CircleAvatar(
+    final userAvatar = CircleAvatar(
       child: Stack(
         children: [
           const Center(child: Text("PR")),
```

Comment:
```suggestion
          const Center(child: Text("PR")), //TODO: get real initials
```

Implementing it is not in the scope of your PR, but I think it would be good to have the correct initials.
Could add a TODO so we don't forget to fix it? (and I can create an issue in the backlog).
Or quick fix it maybe, if it's easier for you.
@yoannLafore, as scrum master, what do you think?

Reply:
I'll fix it

Reply:
Yeah, never mind this has nothing to do with this PR in reality, I misunderstood the problem you were pointing at first.

I will create an issue in the backlog about have a clear component for the user profile avatar that we can reuse throughout the app.

Reply:
see #157